services/integration-content-repository/app/api/swagger/swagger.json

```diff
@@ -14,10 +14,6 @@
     }
   },
   "tags": [
-    {
-      "name": "IAM",
-      "description": "Identity management"
-    },
     {
       "name": "flows",
       "description": "Operations about flows",
```

Comment:
I would rather call it `Flow management` instead of `Operations about flows`

Reply:
done